File: content/factory-drive/usr/share/examples/hello.go
// Your first Go program. Copy it home, then:   goc hello.go   and   ./hello.wasm
package main

import "kiddos"

func main() {
	kiddos.Println("Hello from Go!")
	kiddos.Print("The screen is ")
	kiddos.PrintInt(kiddos.Cols())
	kiddos.Print(" by ")
	kiddos.PrintInt(kiddos.Rows())
	kiddos.Println(" letters.")
}

File: content/factory-drive/usr/share/examples/hello.c
/* Your first C program. Copy it home, then:   cc hello.c   and   ./hello.wasm */
#include "kiddos.h"

int main(void) {
    kd_print("Hello from C!\n");
    kd_print("The screen is ");
    kd_print_int(kd_cols());
    kd_print(" by ");
    kd_print_int(kd_rows());
    kd_print(" letters.\n");
    return 0;
}
